spec/spec_helper.rb

```diff
@@ -1,5 +1,7 @@
 require "dotenv/load"
 require "simplecov"
+require "active_support"
+require "active_support/testing/time_helpers"
```

Comment:
I needed this, but it didn't work without also adding the `require "active_support"` line above. We should probably revisit why this is now necessary.